sys-apps/pv/pv-1.6.19-r1.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://www.ivarch.com/programs/sources/${P}.tar.bz2"
 
```

Comment:
I think we do not need to downgrade pv to 1.6.19, as we already have `sys-apps/portage` 3.0.28, right?

Reply:
Oh yeah, I can update that now. Thanks for spotting.

Reply:
I actually had two commits updating `sys-apps/pv`. First was updating it to recent version, the second was downgrading it a bit. Dropped the latter.